package.json

```diff
@@ -17,7 +17,7 @@
     "test-cover": "c8 --check-coverage --branches 100 --functions 100 --lines 100 --statements 100 tape test/markdownlint-test.js test/markdownlint-test-custom-rules.js test/markdownlint-test-helpers.js test/markdownlint-test-result-object.js test/markdownlint-test-scenarios.js",
     "test-declaration": "cd example/typescript && tsc && node type-check.js",
     "test-extra": "node test/markdownlint-test-extra.js",
-    "lint": "eslint --max-warnings 0 lib helpers test schema && eslint --env browser --global markdownit --global markdownlint --rule \"no-unused-vars: 0, no-extend-native: 0, max-statements: 0, no-console: 0, no-var: 0, unicorn/prefer-add-event-listener: 0, unicorn/prefer-query-selector: 0, unicorn/prefer-replace-all: 0\" demo && eslint --rule \"no-console: 0, no-invalid-this: 0, no-shadow: 0, object-property-newline: 0, node/no-missing-require: 0, node/no-extraneous-require: 0\" example",
```

Comment:
Wait, did we lose max-warnings with this change?

Reply:
Ah, yeah, there is no config version of that, it's CLI only. I can add it back, but it will apply to all files being linted. That doesn't seem to be an issue right now as there are no warnings

Reply:
OK, added back to the CLI call